drivers/serial/Kconfig.serial_shell

```
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+# SPDX-License-Identifier: Apache-2.0
+
+config SERIAL_SHELL
+	bool "serial shell"
```

Nit: start with a capital to keep it consisten.

```suggestion
	bool "Serial shell"
```